fly/integration/hijack_test.go

```diff
@@ -874,6 +881,50 @@ var _ = Describe("Hijacking", func() {
 			})
 		})
 
+		Context("when called without specifying path", func() {
+			// var hijacked2 <-chan struct{}
+			BeforeEach(func() {
+				hijackHandlerError = []string{"something went wrong"}
+				hijackExeNotFound = true
+				containerArguments = "build_id=2&step_name=some-step"
+				stepType = "task"
+				stepName = "some-step"
+				buildID = 2
+				path = "bash"
+			})
+			JustBeforeEach(func() {
+				didHijack := make(chan struct{})
+				// hijacked2 = didHijack
+
+				atcServer.AppendHandlers(
+					hijackHandler("container-id-1", didHijack, nil, false, hijackTeamName, "sh"),
+				)
+			})
+			It("tries bash then sh when hijack returns executable not found error", func() {
+				flyCmd := exec.Command(flyPath, "-t", targetName, "hijack", "-b", "2", "-s", "some-step")
+
+				stdin, err := flyCmd.StdinPipe()
+				Expect(err).NotTo(HaveOccurred())
+
+				sess, err := gexec.Start(flyCmd, GinkgoWriter, GinkgoWriter)
+				Expect(err).NotTo(HaveOccurred())
+
+				Eventually(hijacked).Should(BeClosed())
+				// Eventually(hijacked2).Should(BeClosed())
```

Comment:
I don't really understand how these tests work, so my attempts so far at just copy-pasting things around has only gotten me so far. This test is currently failing because one of the `hijackHandler` mocks is expecting `"some stdin"` but just getting a closed stdin. If I add a second line of
```golang
fmt.Fprintf(stdin, "some stdin")
```
then the test with expecting `"some stdin"` but getting `"some stdinsome stdin"`.

I am pretty stumped on how to resolve this...